File: app/src/main/java/vergecurrency/vergewallet/viewmodel/ReceiveFragmentViewModel.java
package vergecurrency.vergewallet.viewmodel;

import androidx.lifecycle.ViewModel;
import vergecurrency.vergewallet.wallet.WalletManager;

public class ReceiveFragmentViewModel extends ViewModel {

	private String receiveAddress;

	public ReceiveFragmentViewModel() {
		receiveAddress = WalletManager.getReceiveAddress();
	}

	public String getReceiveAddress() {
		return receiveAddress;
	}

	public void setReceiveAddress(String receiveAddress) {
		this.receiveAddress = receiveAddress;
	}

	public void newReceiveAddress() {
	}
}

File: app/src/main/kotlin/vergecurrency/vergewallet/viewmodel/ReceiveFragmentViewModel.kt
package vergecurrency.vergewallet.viewmodel

import androidx.lifecycle.ViewModel
import vergecurrency.vergewallet.wallet.WalletManager

class ReceiveFragmentViewModel : ViewModel() {

    var receiveAddress: String? = null

    init {
        receiveAddress = WalletManager.receiveAddress
    }

    fun newReceiveAddress() {}
}
